tests/engine/test_state.py

```diff
@@ -108,6 +108,11 @@ def test_scheduled_states_have_default_times():
     assert now - Retrying().start_time < datetime.timedelta(seconds=0.1)
 
 
+def test_queued_states_have_default_times():
```

Comment:
```suggestion
def test_queued_states_have_start_times():
```
Sorry, just a minor test name change suggestion!